lib/flipper/adapter.rb

```diff
@@ -43,6 +43,13 @@ def get_multi(features)
       result
     end
 
+    # Public: Does a feature with this key exist.
+    #
+    # Returns true if a feature exists with this key, else false.
+    def exist?(key)
+      features.include?(key)
```

Comment:
Add a default implementation on the base adapter, that we can override in the Redis adapter